content/library/advanced-features/app-testing/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Streamlit's native app testing framework
+slug: /library/advanced-features/app-testing
+---
+
+# Streamlit's native app testing framework
+
+Streamlit app testing enables developers to build and run automated tests. Bring your favorite test automation software and enjoy simple syntax to simulate user input and inspect rendered output.
+
+The provided class, AppTest, simulates a running app and provides methods to set up, manipulate, and inspect the app contents via API instead of a browser UI. AppTest provides similar functionality to browser automation tools like Selenium or Playwright, but with less overhead to write and execute tests. Use our testing framework with a tool like [pytest](https://docs.pytest.org/) to execute or automate your tests. A typical pattern is to build a suite of tests for an app to ensure consistent functionality as the app evolves. The tests run locally and/or in a CI environment like Github Actions.
```

Comment:
nit: `Github` -> `GitHub`

Reply:
✅ In the proofread edits I just commited.